docs/developer/cmds.md

```diff
@@ -2,7 +2,7 @@
 
 The following utilities, defined in the `cmd/` directory, are useful for
 development and testing. They **should not** be relied on for production
-purposes. In particular, command line options and default behaviour can change
+purposes. In particular, command line options and default behavior can change
```

Comment:
😢 😭

Reply:
Not sure if we should argue about
```
matrix:
    aspell:
      d: en_US
```
😉

Reply:
Not arguing, just hurts my soul a little 😆😭